### DISCUSSION TITLE
install/community/resources

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3563] [Announcements]
# install/community/resources

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/community/resources.html

**Comment by samzong** [comment]
这里推荐的 k8s 版本需要更新了。。。

**Comment by windsonsea** [comment]
Fixed in PR #5289
